central/graphql/resolvers/vulnerability_requests_test.go

```diff
@@ -207,16 +198,31 @@ func (s *VulnRequestResolverTestSuite) SetupTest() {
 		AuditLogger:    audit.New(notifierMock),
 	}
 
+	var err error
 	s.schema, err = graphql.ParseSchema(Schema(), s.resolver)
 	s.NoError(err)
```

Comment:
Change this to avoid assigning to a member field before checking error.

```suggestion
	schema, err := graphql.ParseSchema(Schema(), s.resolver)
	s.NoError(err)
	s.schema = schema
```